game.js

```diff
@@ -54,8 +54,8 @@
     ) {
       playerScore++;
     }
-    compScore.textContent = "computer score: " + computerScore;
-    yourScore.textContent = "player score: " + playerScore;
+    compScore.textContent = computerScore + " :";
```

Comment:
Appending " :" works. A little cleaner solution would be to define the ":" in HTML outside of the computer score container.